schema.sql

```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE nearbyHomes;
```

Comment:
you could also add IF NOT EXISTS it only creates the database or table if they don't exist

Reply:
thanks! i'll add this in